### PR TITLE
Set Portable PDBs in Android and iOS project templates

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type
What kind of change does this PR introduce?
Enhancement

## What is the current behavior?
Android and iOS may fail to break on breakpoints in `.xaml.cs` if the debugging symbol type is Full in projects created from templates.

## What is the new behavior?
Breakpoints work in `.xaml.cs` files

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)

### CODE REVIEW

File: src/SolutionTemplate/UnoSolutionTemplate/UnoQuickStart.Droid/UnoQuickStart.Droid.csproj [hunk 1]
```diff
@@ -23,7 +23,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
+    <DebugType>portable</DebugType>
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/UnoSolutionTemplate/UnoQuickStart.Droid/UnoQuickStart.Droid.csproj]: I was impressed with your diagnosis on Gitter. Why is this necessary, though?

Comment (jeromelaban) [reply src/SolutionTemplate/UnoSolutionTemplate/UnoQuickStart.Droid/UnoQuickStart.Droid.csproj]: Thanks ;) It's related to the recent changes with the mono debugger, where sequence points of the mdb format were not handling partial classes properly, as well as lambdas. Portable PDBs do.
